scripts/mirror_file.py

```diff
@@ -31,6 +32,12 @@
 from azul.logging import (
     configure_script_logging,
 )
+from azul.plugins import (
+    File,
+)
+from azul.plugins.metadata.hca import (
+    HCAFile,
+)
```

Comment:
This smells, since the concrete implementation of the new `File` interface should be a plugin detail.

However, I don't know if this is a smell already on `develop` or how difficult it would be to fix. Please request a PL slot about this.

Reply:
Agreed at PL to pull up the `content_type` field into the superclass